src/test/java/StringTest.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+    @Test
+    @DisplayName("String.split() 에 대한 학습 테스트")
+    void splitMethodTest(){
+        String numberList = "1,2";
+        String singleNumber = "1";
+
+        String[] numberListResult = numberList.split(",");
+        String[] singleNumberResult = singleNumber.split(",");
+
+        assertThat(numberListResult).contains("1","2");
```

Comment:
```suggestion
        assertThat(numberListResult).containsExactly("1", "2");
```

오히려 2개 이상의 값일 때 containsExactly으로 확인하는 것이 의미있지 않을까?
추가로 space convention 위반